src/app/components/experiment-workspace/experiment-workspace.component.ts

```diff
@@ -103,8 +103,8 @@ export class ExperimentWorkspaceComponent implements OnInit, OnDestroy {
             return null;
         }
         const time = new Date();
-        
-        const timelinePage = this.backend.getTimelineStepsPage(this.experimentId, num, this.stepsPerPage)        
+
+        const timelinePage = this.backend.getTimelineStepsPage(this.experimentId, num, this.stepsPerPage, undefined, undefined, undefined, undefined, undefined); // TODO: add other values?        
```

Comment:
you can just ignore parameters at the end of the list. Javascript will fill up parameters with undefined on its own.